presto-tests/src/test/java/io/prestosql/memory/TestMemoryManager.java

```diff
@@ -327,6 +329,50 @@ public void testClusterPools()
         }
     }
 
+    @Test(timeOut = 240_000)
+    public void testSessionQueryMemoryPerNodeLimit()
+            throws Exception
+    {
+        try (DistributedQueryRunner queryRunner = createQueryRunner(TINY_SESSION, ImmutableMap.of())) {
```

Comment:
please share query runner across different tests. Query runner takes time to create

Reply:
I tried extending AbstractTestQueryFramework but it seems to be a generic framework with a lot of tests which are not applicable here.
Furthermore, all the tests set properties while creating query runner so they require creation of different query runners.

Reply:
`AbstractTestQueryFramework` doesn't have any tests. You can use methods that accept `Session` as it's parameter, e.g. `io.prestosql.tests.AbstractTestQueryFramework#assertQuery(io.prestosql.Session, java.lang.String, java.lang.String)`